docs/web3.miner.rst

```diff
@@ -71,23 +88,33 @@ The following methods are available on the ``web3.geth.miner`` namespace.
         >>> web3.geth.miner.stop()
 
 
-.. py:method:: GethMiner.startAutoDAG()
+.. py:method:: GethMiner.start_auto_dag()
 
     * Delegates to ``miner_startAutoDag`` RPC Method
 
     Enable automatic DAG generation.
 
     .. code-block:: python
 
-        >>> web3.geth.miner.startAutoDAG()
+        >>> web3.geth.miner.start_auto_dag()
 
+.. py:method:: GethMiner.startAutoDAG()
```

Comment:
This is existing, but we only have a `startAutoDag` method, not `startAutoDAG`. Do you mind changing?
```suggestion
.. py:method:: GethMiner.startAutoDag()
```